project-base/src/Twig/ImageExtension.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Why is this happening in project-base and not in framework?

Reply:
This should be mentioned as deprecated right? So in next major this code would be moved to framework. Same for services.yaml

Reply:
You're right! fixed in 73e9ce93a

I did not add trigger error deprecation as it's no way, how a user can solve the deprecation.